src/Normalizer/JecouteSurveyNormalizer.php

```diff
@@ -27,12 +27,18 @@ public function normalize($object, $format = null, array $context = [])
 
         $data['questions'] = array_map(function (SurveyQuestion $surveyQuestion) use ($format, $context) {
             $question = $surveyQuestion->getQuestion();
+            $choices = $this->normalizer->normalize($question->getChoices(), $format, $context);
+
+            if (isset($context['item_operation_name']) && 'put' === $context['item_operation_name']) {
+                $choices = $this->normalizer->normalize($question->getChoicesOrdered(), $format, $context);
+                $choices = array_values($choices);
+            }
```

Comment:
what about doing this without IF: 
```suggestion
            $choices = $this->normalizer->normalize($question->getChoicesOrdered(), $format, $context);
```